data/extension/init/templates/tinygo/envoy.access_loggers/default/go.sum

```diff
@@ -0,0 +1,13 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
I don't mind excluding them at all. Just curious, What's the reason behind it?

Reply:
In any way, I'm going to remove `.sum`s to be aligned wit Rust

Reply:
OK, added the `go.sum` since TinyGo seem not able to generate go.sum properly and incur some nasty errors. I don't think adding them is not harmless. If an issue comes up, let's come back later.